pkg/utils/validation/cidr/disjoint.go

```diff
@@ -49,12 +47,8 @@ func validateOverlapWithSeed(fldPath *field.Path, shootNetwork *string, networkT
 			allErrs = append(allErrs, field.Invalid(fldPath, *shootNetwork, fmt.Sprintf("shoot %s network intersects with seed node network", networkType)))
 		}
 
-		if NetworksIntersect(v1beta1constants.DefaultVPNRange, *shootNetwork) {
-			allErrs = append(allErrs, field.Invalid(fldPath, *shootNetwork, fmt.Sprintf("shoot %s network intersects with default vpn network (%s)", networkType, v1beta1constants.DefaultVPNRange)))
-		}
-
-		if NetworksIntersect(v1beta1constants.DefaultVPNRangeV6, *shootNetwork) {
-			allErrs = append(allErrs, field.Invalid(fldPath, *shootNetwork, fmt.Sprintf("shoot %s network intersects with default vpn network (%s)", networkType, v1beta1constants.DefaultVPNRangeV6)))
+		if seedVPN != nil && NetworksIntersect(*seedVPN, *shootNetwork) {
+			allErrs = append(allErrs, field.Invalid(fldPath, *shootNetwork, fmt.Sprintf("shoot %s network intersects with seed vpn network", networkType)))
```

Comment:
VPN in caps? This also needs to be adapted in the tests.
```suggestion
			allErrs = append(allErrs, field.Invalid(fldPath, *shootNetwork, fmt.Sprintf("shoot %s network intersects with seed VPN network", networkType)))
```